collector/collector.go

```diff
@@ -56,9 +58,11 @@ func (c *eventsCollector) collectEvents(url string) {
 	atxs, err := sub.Subscribe(events.EventNewAtx)
 	atxsValid, err := sub.Subscribe(events.EventAtxValid)
 	reward, err := sub.Subscribe(events.EventRewardReceived)
+	created, err := sub.Subscribe(events.EventCreatedBlock)
+	createdAtx, err := sub.Subscribe(events.EventCreatedAtx)
 	sub.StartListening()
 	if err != nil {
-		log.Info("cannot start subscriber")
+		log.Debug("cannot start subscriber")
```

Comment:
general note: it would be better if you created a named logger for the collector instead of using the logger.

Reply:
this is used only for debug and there should be only one collector per system, so naming this logger is not that important